### ISSUE TITLE
DomainNet Experimental Settings

### ISSUE THREAD

**Issue by realKuoShi** [LeapLabTHU/DAPrompt#4]
Hi John,

Fantastic work you have done!

I see you add an additional dataset experiment on this repo, that's the DomainNet which is not included in the paper.

May I have the Experimental Settings for the DomainNet? For example the training epoch, batch size, pseudo-labeling threshold and etc.

**Comment by John-Ge** [comment]
We use batch size 32, learning rate 3e-3 and train the model for 15 epoch. You can try other hyper-parameters.
